massgov/openmass
# DP-23089: Add parent field to most node types

**Description:**
Added the "Parent page" and "Short title" fields to content types.

**Jira:** (Skip unless you are MA staff)
DP-23089

**To Test:**
- Login and edit the following content types:
- Advisory, Binder, Curated List, Decision, Decision Tree, Event, Executive Order, Form, Guide, How-to, Information Details, Location, Location Detail, News, Organization, Person, Promo Page, Regulation, Rules of Court, Service, Service Details, Topic page
- Verify there is a “Parent page” field before the “Organization(s)” field.
- Verify there is no attached “Weight” field with the “Parent page” field.
- Verify the field is required.
- Verify the help text matches: “Choose the parent for the current page. Parents are higher-level pages that link to child pages that contain more detailed content. A link to the parent will display in the bread crumb element once it's fully implemented. Normally, you will want to make sure that you have a link on the parent to this page.”
- Verify there is a “Children” tab.
- Create the following content types:
- Advisory, Binder, Curated List, Decision, Decision Tree, Event, Executive Order, Form, Guide, How-to, Information Details, Location, Organization, Promo Page, Regulation, Rules of Court, Service, Service Details, Topic page
- Verify there is a “Short title” field after the “Title” field.
- Verify the field character limit is 70.
- Verify the field is optional.
- Verify the help text matches: “The short title will be used in the breadcrumb if this page is the parent of another page. If left blank, the title for this page will be used in the breadcrumb instead. Because the short title is only used in the breadcrumb, you can assume that visitors will have the context of the current page title and the breadcrumb elements. If you can, try to shorten the main title first. Avoid abbreviations.”

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
